cs/src/Connections/TunnelClient.cs

```diff
@@ -554,7 +563,6 @@ public override async ValueTask DisposeAsync()
         if (this.SshSession != null)
         {
             await this.SshSession.CloseAsync(SshDisconnectReason.ByApplication);
-            this.SshSession.Dispose();
```

Comment:
Why? Is this redundant?

Reply:
Yes it's redundant. `CloseAsync` [will dispose the session](https://github.com/microsoft/dev-tunnels-ssh/blob/96dfe16e8d4cf1ddbe2dae52560c60899a87d133/src/cs/Ssh/SshSession.cs#L658).